core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/directives/dot-experiment-trim-input/dot-experiments-trim-input.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, HostListener } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+/**
+ * Directive to trim input value on blur event.
+ */
+@Directive({
+    selector: '[dotTrimInput]',
+    standalone: true
+})
+export class DotExperimentsTrimInputDirective {
```

Comment:
If this directive can be used in other places besides the experiment portlet, should we rename the directive to 'dotTrimInputDirective' and move it to `ui` lib?

Reply:
Yes. Good idea.